apps/web/lib/actions/partners/onboard-partner.ts

```diff
@@ -60,6 +60,7 @@ export const onboardPartnerAction = authUserActionClient
           create: {
             userId: user.id,
             role: "owner",
+            notificationPreference: {},
```

Comment:
A `PartnerNotificationPreferences` record will be created when the partner is created, but we'll have to backfill this for existing partners.